src/components/Nav/Nav.tsx

```diff
@@ -196,6 +194,22 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
     ev.preventDefault();
     ev.stopPropagation();
   }
+
+  private _onNavAnchorLinkClicked(link: INavLink, ev: React.MouseEvent): void {
+    this._selectedKey = link.key;
+
+    if (this.props.onLinkClick) {
+      this.props.onLinkClick(ev, link);
+    }
+  }
+
+  private _onNavButtonLinkClicked(link: INavLink, ev: React.MouseEvent): void {
+    this._selectedKey = link.key;
+
+    if (link.onClick) {
+      link.onClick(ev);
```

Comment:
Seemed there was a commit override that took out a commit that did this already. Changed!